napari/components/viewer_model.py

```diff
@@ -518,6 +558,9 @@ def _on_remove_layer(self, event):
         self._on_layers_change()
         self._on_grid_change()
 
+        if len(self.layers) == 1:
```

Comment:
I don't understand why this only applies when the number of layers is 1.

In general, I would expect that removing a layer could always change the point and thus require us to update all the layers.

Reply:
Yeah most likely. let's always update the layers on layer remove.